### ISSUE TITLE
Plotting a quasimatrix with columns that have delta functions crashes.

### ISSUE THREAD

**Issue by aaustin141** [chebfun/chebfun#1316]
It seems we can't plot quasimatrices that have columns with delta functions:
```
>> x = chebfun(@(x) x);
>> Q = [dirac(x) dirac(x)];
>> plot(Q)
Error using stem (line 41)
Too many input arguments.
Error in chebfun/plot>mystem (line 446)
    h(j) = stem(data{:}, 'fill');
Error in chebfun/plot (line 358)
    h4 = mystem(deltaData{:}); 
```

**Comment by asgeirbirkis** [comment]
This works for me now without crashing, @aaustin141 can you please confirm you're happy for this to be closed?

**Comment by nickhale** [comment]
This doesn't crash for me, but I notice that the colours are off (at least, in R2014b):
```
>> Q = [dirac(x+.5)+exp(x) dirac(x-.5)-sin(x) dirac(x)+1.5];
>> plot(Q, 'LineWidth', 2)
```
![plotdelta](https://cloud.githubusercontent.com/assets/2035010/6634786/0ef70372-c966-11e4-9c46-a3c727545e33.png)

**Comment by aaustin141** [comment]
I'm getting the same behavior as @nickhale:  the original example no longer crashes, but the colors are messed up.  We should be able to fix this by messing with `ColorOrderIndex` in the same way that we used to fix #1263 via #1278.
